src/Pretzel.Logic/Commands/BuiltInCommands.cs

```diff
@@ -0,0 +1,13 @@
+namespace Pretzel.Logic.Commands
+{
+    public static class BuiltInCommands
+    {
+        public const string Bake = "bake";
```

Comment:
Sorry, one last nitpick: I am not sure about the need for these constants, are they used internally and I have missed their usage?
Because for the extensibility, all go through types, right?

Reply:
Those were from the first round where we used string based mapping instead of types. Not used -> deleted.